apis/dscinitialization/v1alpha1/dscinitialization_types.go

```diff
@@ -32,8 +32,10 @@ const (
 
 // DSCInitializationSpec defines the desired state of DSCInitialization
 type DSCInitializationSpec struct {
-	Namespaces   []string   `json:"namespaces"`
+	// +kubebuilder:default:=opendatahub
+	Namespace    string     `json:"namespace"`
```

Comment:
Should this be named as applicationsNamespace?

Reply:
sure i can rename it

Reply:
fixed and updated PR description.